data/users/externalldap.py

```
@@ -381,76 +554,214 @@ def _build_user_information(self, response):
+                                    "ldap.filter": search_flt,
+                                    "ldap.paged": True,
+                                    "ldap.serverctrls": str(lc),
+                                },
+                            ) as sspan:
+                                msgid = conn.search_ext(
+                                    user_search_dn,
+                                    ldap.SCOPE_SUBTREE,
+                                    search_flt,
+                                    serverctrls=[lc],
+                                )
+                                _, rdata, _, serverctrls = conn.result3(msgid)
```

Ruff (0.14.4)

Unpacked variable `serverctrls` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)